testplan/testing/multitest/base.py

```diff
@@ -1033,7 +1032,7 @@ def _run_testcases_iter(self, testsuite, testcases):
         """
         pre_testcase = getattr(testsuite, "pre_testcase", None)
         post_testcase = getattr(testsuite, "post_testcase", None)
-        testsuite_uid = testsuite.name
+        testsuite_uid = testsuite.name  # Unique suite name in Multitest as UID
```

Comment:
I think it is confusing to assign testsuite.name to another local variable called testsuite_uid. Can we simply use testsuite.name where it is needed?

Reply:
I think the author want to emphasize that unique id is used as identifier when transferred through network. let me think about it again.